### DISCUSSION TITLE
2.1.0-beta.1

### DISCUSSION THREAD

**Discussion by joschi** [OpenAPITools/openapi-diff#389] [Announcements]
## What's Changed
* Migrate from Log4j 1.2.x to Logback by @joschi in https://github.com/OpenAPITools/openapi-diff/pull/307
* Exclude null properties when rendering JSON by @gloeglm in https://github.com/OpenAPITools/openapi-diff/pull/302
* Remove dependency on commons-httpclient by @joschi in https://github.com/OpenAPITools/openapi-diff/pull/308
* Use SLF4J commons-logging bridge by @joschi in https://github.com/OpenAPITools/openapi-diff/pull/311
* build(deps): bump maven-site-plugin from 3.9.1 to 3.10.0 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/313
* build(deps): bump assertj-core from 3.21.0 to 3.22.0 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/315
* Update README.md by @seguri-doodle in https://github.com/OpenAPITools/openapi-diff/pull/324
* build(deps): bump maven-compiler-plugin from 3.8.1 to 3.10.0 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/333
* build(deps): bump slf4j.version from 1.7.32 to 1.7.33 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/323
* build(deps): bump maven-jar-plugin from 3.2.0 to 3.2.2 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/322
* build(deps): bump maven-plugin-plugin from 3.6.2 to 3.6.4 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/340
* build(deps): bump nexus-staging-maven-plugin from 1.6.8 to 1.6.12 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/339
* build(deps): bump maven-javadoc-plugin from 3.3.1 to 3.3.2 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/341
* build(deps): bump maven-site-plugin from 3.10.0 to 3.11.0 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/342
* build(deps): bump slf4j.version from 1.7.33 to 1.7.36 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/338
* Update Docker release action with multi-platform by @kaschula in https://github.com/OpenAPITools/openapi-diff/pull/336
* build(deps): bump swagger-parser.version from 2.0.28 to 2.0.30 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/337
* fix: mixed up logic around response mediatype addition/removal by @slinstaedt in https://github.com/OpenAPITools/openapi-diff/pull/328
* #198: Fixing false positive breaking change reported when removing an optional field from a response by @orange-buffalo in https://github.com/OpenAPITools/openapi-diff/pull/327
* Support allOf schemas referring to oneOf schemas by @julienrf in https://github.com/OpenAPITools/openapi-diff/pull/318
* build(deps): bump actions/checkout from 2.4.0 to 3 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/343
* upgrade m-release-p: outputTimestamp auto-update by @hboutemy in https://github.com/OpenAPITools/openapi-diff/pull/314
* Equivalent paths with overloaded parameters are supported by @timur27 in https://github.com/OpenAPITools/openapi-diff/pull/306
* Fix stackoverflow in recursive definitions by @mikkela in https://github.com/OpenAPITools/openapi-diff/pull/331
* build(deps): bump logback-classic from 1.2.10 to 1.2.11 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/347
* build(deps): bump maven-compiler-plugin from 3.10.0 to 3.10.1 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/350
* build(deps): bump swagger-parser.version from 2.0.30 to 2.0.31 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/354
* build(deps): bump actions/cache from 2.1.7 to 3 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/356
* build(deps): bump maven-shade-plugin from 3.2.4 to 3.3.0 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/360
* build(deps): bump swagger-parser.version from 2.0.31 to 2.0.32 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/363
* build(deps): bump jacoco-maven-plugin from 0.8.7 to 0.8.8 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/365
* build(deps): bump actions/setup-java from 2 to 3 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/367
* build(deps): bump maven-site-plugin from 3.11.0 to 3.12.0 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/368
* build(deps): bump maven-javadoc-plugin from 3.3.2 to 3.4.0 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/369
* build(deps): bump nexus-staging-maven-plugin from 1.6.12 to 1.6.13 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/370
* build(deps): bump docker/login-action from 1 to 2 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/374
* build(deps): bump docker/setup-buildx-action from 1 to 2 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/373
* build(deps): bump docker/build-push-action from 2 to 3 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/372
* build(deps): bump swagger-parser.version from 2.0.32 to 2.0.33 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/376
* build(deps): bump assertj-core from 3.22.0 to 3.23.0 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/377
* build(deps): bump assertj-core from 3.23.0 to 3.23.1 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/378
* Add support for remote development with Gitpod by @joschi in https://github.com/OpenAPITools/openapi-diff/pull/379
* Generate SBOM using CycloneDX Maven plugin by @joschi in https://github.com/OpenAPITools/openapi-diff/pull/382
* Bump Maven to version 3.8.5 and Maven wrapper to version 3.1.1 by @joschi in https://github.com/OpenAPITools/openapi-diff/pull/383
* fix #358 - correction of regex problem + tests + auto formatting by @dedece35 in https://github.com/OpenAPITools/openapi-diff/pull/359
* build(deps): bump maven-release-plugin from 3.0.0-M5 to 3.0.0-M6 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/385
* build(deps): bump swagger-parser.version from 2.0.33 to 2.1.0 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/386
* build(deps): bump swagger-parser.version from 2.1.0 to 2.1.1 by @dependabot in https://github.com/OpenAPITools/openapi-diff/pull/387
* Fix: Crashes when parsing diff for very large specs by @eacolina in https://github.com/OpenAPITools/openapi-diff/pull/388

## New Contributors
* @gloeglm made their first contribution in https://github.com/OpenAPITools/openapi-diff/pull/302
* @seguri-doodle made their first contribution in https://github.com/OpenAPITools/openapi-diff/pull/324
* @kaschula made their first contribution in https://github.com/OpenAPITools/openapi-diff/pull/336
* @slinstaedt made their first contribution in https://github.com/OpenAPITools/openapi-diff/pull/328
* @orange-buffalo made their first contribution in https://github.com/OpenAPITools/openapi-diff/pull/327
* @julienrf made their first contribution in https://github.com/OpenAPITools/openapi-diff/pull/318
* @hboutemy made their first contribution in https://github.com/OpenAPITools/openapi-diff/pull/314
* @timur27 made their first contribution in https://github.com/OpenAPITools/openapi-diff/pull/306
* @mikkela made their first contribution in https://github.com/OpenAPITools/openapi-diff/pull/331
* @dedece35 made their first contribution in https://github.com/OpenAPITools/openapi-diff/pull/359
* @eacolina made their first contribution in https://github.com/OpenAPITools/openapi-diff/pull/388

**Full Changelog**: https://github.com/OpenAPITools/openapi-diff/compare/2.0.1...2.1.0-beta.1

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenAPITools/openapi-diff/releases/tag/2.1.0-beta.1'>2.1.0-beta.1</a>.</em>

**Comment by jamiecuthill** [comment]
@joschi Any idea on the timeline to fix the [docker release](https://github.com/OpenAPITools/openapi-diff/actions/runs/2565520017)?

**Reply by joschi** [reply]
@jamiecuthill I currently don't have time to work on this but I'd be happy to merge a PR which fixes the container image build and trigger another beta release.
